game/script-idles.rpy

```diff
@@ -40,6 +40,7 @@ init python in jn_idles:
         gaming = 2
         resting = 3
         vibing = 4
+        working_on_papers = 5
```

Comment:
Sorry, I don't think I was very clear here! I was referring to using `working_on_papers` as the new animation name for Nat - this would be fine as just `working`, so we can keep it generic.

Reply:
Ah, I did notice what you meant, but felt like it would be better to keep it consistent as the other idles are the same name like animation as well.